src/ui/public/kbn_share/kbn_share.less

```diff
@@ -0,0 +1,42 @@
+.sharing-ui-menu__List {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.sharing-ui-menu__ListWrapper,
+.sharing-ui-menu__SharePanel {
```

Comment:
These styles look great; I would just recommend sticking to all snake-case or all camel-case for consistency, instead of mixing them.